src/coreclr/src/System.Private.CoreLib/src/System/RuntimeType.CoreCLR.cs

```diff
@@ -1003,31 +1003,16 @@ private RuntimeType[] PopulateInterfaces(Filter filter)
 
                             Type[] temp = constraint.GetInterfaces();
                             for (int j = 0; j < temp.Length; j++)
-                                al.Add(temp[j] as RuntimeType);
+                                al.Add((temp[j] as RuntimeType)!);
```

Comment:
```suggestion
                                al.Add((RuntimeType)temp[j]);
```
to avoid `!` ?

Reply:
Yeah, I never know when I see `as` used in code like this whether that was a stylistic choice or it actually might be something other than the target.  If it's definitely going to be a RuntimeType, I'd certainly prefer the explicit cast rather than `as`.